tools/doc/html.mjs

```diff
@@ -22,7 +22,7 @@
 import fs from 'fs';
 import path from 'path';
 
-import highlightJs from 'highlight.js';
+import highlightJs from 'highlight.js/lib/core';
```

Comment:
Argh, I think we're not registering the language strings so this may mean that I've disabled highlighting altogether? That would not be great. Checking....

Reply:
Yup, this disabled all highlighting. I need to add another line or few lines of code....

Reply:
Although as a bonus, this gives me a way to turn off `bash` highlighting for the many times `bash` is used for CLI stuff in our docs. So that's a plus from my perspective.

Reply:
OK fixed.